### PR TITLE
[10.0][FIX] mail_base & mail_create_partner

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Fixes.

### CODE REVIEW

File: mail_create_partner/static/src/js/mail_create_partner.js [hunk 1]
```diff
@@ -31,14 +32,15 @@ odoo.define('mail_create_partner.create_partner_object', function (require) {
...
             };
+            this.message_id = message_id;
 
-            self.do_action(action, {
+            this.do_action(action, {
                 'on_close': function(){
-                    var message = base_obj.chat_manager.get_message(self.message_id);
-                    chat_manager.bus.trigger('update_message', message);
-                    self.fetch_and_render_thread();
+//                    var message = base_obj.chat_manager.get_message(this.message_id);
```
Comment (AaronHForgeFlow) [inline mail_create_partner/static/src/js/mail_create_partner.js:41]: why not delete?

Comment (MiquelRForgeFlow) [reply mail_create_partner/static/src/js/mail_create_partner.js:41]: It acts as a reminder.

Comment (AaronHForgeFlow) [reply mail_create_partner/static/src/js/mail_create_partner.js:41]: is it a To do?

Comment (MiquelRForgeFlow) [reply mail_create_partner/static/src/js/mail_create_partner.js:41]: No. It's nothing. It's just I didn't want to delete. May serve as hint if in the future needs an improvement.

**Review by AaronHForgeFlow — COMMENTED** [review]
Please check travis.
Functionally it works :+1:

**Pull request merged**